### PR TITLE
[RFR] Add function scoped ansible cat item fixture

### PR DESCRIPTION
**Pull request by mshriver**
The global fixture is module scoped, and the tests in test_embedded_ansible_services are modifying the object given by the fixture, breaking other tests.

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 0fa407c037d891bc35800ecf6db9d2b4d4c3a15b

The *local* fixture **`local_ansible_catalog_item`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_linked_vm_action*
    - *custom_service_button*
    - *test_ansible_service_cloud_credentials*

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`custom_service_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by mshriver** [comment]
Waiting to restart PRT after @izapolsk re-deploys it.

**Comment by mshriver** [comment]
@ManageIQ/qe-triage please review (testing)

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 1]
```diff
@@ -46,14 +46,45 @@
 ]
 
 
+@pytest.fixture(scope="function")
+def local_ansible_catalog_item(appliance, ansible_repository):
+    """override global ansible_catalog_item for function scope
+        as these tests modify the catalog item
```
Comment (john-dupuy) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: Can I still resolve my comments? (test)

Comment (mshriver) [reply cfme/tests/ansible/test_embedded_ansible_services.py]: no (answer)

**Pull request merged**